conformance_tests/sysman/test_sysman_device/src/test_sysman_device.cpp

```diff
@@ -643,5 +651,28 @@ TEST_F(
     }
   }
 }
-
+TEST_F(SYSMAN_DEVICE_TEST,
+       GivenValidDeviceWhenWarmResettingDeviceThenDeviceResetExtIsSucceded) {
```

Comment:
IsSucceded -> Succeeds ?
Please update the other places as well

Reply:
Updated as per the suggestion